language/move-analyzer/tests/symbols/sources/M1.move

```diff
@@ -133,4 +133,4 @@ module Symbols::M1 {
         }
     }
 
-}
+}
```

Comment:
Could we fix this please so that this file is unchanged?

Reply:
Done